### PR TITLE
Recent auctions

### PR DESCRIPTION
**Pull request by matthias-luger**

### CODE REVIEW

File: src/components/AuctionDetails/AuctionDetails.tsx [hunk 1]
```diff
@@ -167,13 +159,15 @@ function AuctionDetails(props: Props) {
...
                 </p>
-                <p style={{ cursor: "pointer" }} onClick={() => navigateToPlayer(auctionDetails?.auctioneer.uuid)}>
-                    <span className="label">
-                        <Badge variant={labelBadgeVariant}>Auctioneer:</Badge>
-                    </span>
-                    {auctionDetails?.auctioneer.name}
-                    <img crossOrigin="anonymous" src={auctionDetails?.auctioneer.iconUrl} alt="" height="16" width="16" style={{ marginLeft: "5px" }} />
-                </p>
+                <Link to={`/player/${auctionDetails.auctioneer.uuid}`}>
+                    <p style={{ cursor: "pointer" }}>
```
Comment (Ekwav) [inline src/components/AuctionDetails/AuctionDetails.tsx:163]: this is an `a` element now the style is not needed anyome

File: src/components/RecentAuctions/RecentAuctions.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@media all and (max-width: 480px) {
+    .cardWrapper {
+        padding: 15px;
+        width: 100%;
+        display: inline-block;
```
Comment (Ekwav) [inline src/components/RecentAuctions/RecentAuctions.css:5]: padding: 15px; and `display: inline-block;` are redundant

Comment (Ekwav) [inline src/components/RecentAuctions/RecentAuctions.css:5]: padding: 15px; and `display: inline-block;` are redundant

File: src/utils/LoadingUtils.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ export function getLoadingElement(): JSX.Element {
 export function getInitialLoadingElement(): JSX.Element {
     return (<div className="center" style={{ height: '500px' }}>
         <div>
-            <img src="/logo192.png" height="192"/>
+            <img src="/logo192.png" height="192" alt=""/>
```
Comment (Ekwav) [inline src/utils/LoadingUtils.tsx:15]: why was this added?

**Review by Ekwav — CHANGES_REQUESTED** [review]

File: src/utils/LoadingUtils.tsx (hunk 3 above)
Comment (matthias-luger) [reply src/utils/LoadingUtils.tsx:15]: because it results in a warning

File: src/components/RecentAuctions/RecentAuctions.css (hunk 2 above)
Comment (matthias-luger) [reply src/components/RecentAuctions/RecentAuctions.css:5]: cant be helped

File: src/components/RecentAuctions/RecentAuctions.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    }
+}
+
+@media all and (min-width: 1024px) {
+    .cardWrapper {
+        padding: 15px;
+        width: 16.66666666666666666666666666666%;
+        display: inline-block;
+    }
+}
```
Comment (Ekwav) [inline src/components/RecentAuctions/RecentAuctions.css:31]:
```suggestion
.cardWrapper {
    padding: 15px;
    display: inline-block;
}

@media all and (max-width: 480px) {
    .cardWrapper {
        width: 100%;
    }
}

@media all and (min-width: 480px) and (max-width: 768px) {
    .cardWrapper {
        width: 50%;
    }
}

@media all and (min-width: 768px) and (max-width: 1024px) {
    .cardWrapper {
        width: 25%;
    }
}

@media all and (min-width: 1024px) {
    .cardWrapper {
        width: 16.66666666666666666666666666666%;
    }
}
```

**Review by Ekwav — CHANGES_REQUESTED** [review]

File: src/utils/LoadingUtils.tsx (hunk 3 above)
Comment (Ekwav) [reply src/utils/LoadingUtils.tsx:15]: please write "logo" in there

**Review by Ekwav — APPROVED** [review]
